src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/LicenceController.kt

```diff
@@ -468,7 +468,7 @@ class LicenceController(
     @Valid @RequestBody
     request: AddAdditionalConditionRequest,
   ): AdditionalCondition {
-    return this.licenceConditionService.addAdditionalCondition(licenceId, conditionType, request)
+    return this.licenceConditionService.addAdditionalCondition(licenceId, request)
```

Comment:
condition type wasn't used - this was is currently part of the request. Ideally we'd remove it entirely and derive from condition code